src/wix/wixnative/smartcab.cpp

```diff
@@ -1,14 +1,15 @@
 // Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information.
 
 #include "precomp.h"
+#include "fileutil.h"
```

Comment:
I think this should have been added to `precomp.h` instead.

Reply:
Actually, I missed it when removing the CI test code I added.